packages/edit-post/README.md

```diff
@@ -48,6 +48,7 @@ Renders a new item in the block settings menu.
 
 _Usage_
 
+<caption>ES5</caption>
```

Comment:
I think it should strip this line with the `<caption>`. If I recall properly it has some special handling in the Block Editor Handbook already.

Reply:
Where did this happen before?

Reply:
@nosolosw should have an answer 😃

Reply:
Did some digging on the `<captions>`. IIRC, the intention was to use them to present tabbed code, like in [this example](https://developer.wordpress.org/block-editor/tutorials/block-tutorial/writing-your-first-block-type/#registering-the-block) for ES5/ESNext. I don't know if we've ever implemented that, though. This is the [issue](https://github.com/WordPress/gutenberg/issues/14331).  Relevant PRs: https://github.com/WordPress/gutenberg/pull/14271 https://github.com/WordPress/gutenberg/pull/14263 https://github.com/WordPress/gutenberg/pull/21405

Additionally, we have to make sure that the comments look good in different places:

- block editor: https://developer.wordpress.org/block-editor/packages/packages-edit-post/#PluginBlockSettingsMenuItem
- npm https://www.npmjs.com/package/@wordpress/edit-post#PluginBlockSettingsMenuItem
- github https://github.com/WordPress/gutenberg/tree/master/packages/edit-post#PluginBlockSettingsMenuItem

I'd think that it's fine to remove the `<caption>` from the comments if they are problematic for `comment-parser`, as I don't see them in use anywhere.